doc/custom-build.md

```diff
@@ -34,28 +34,30 @@ Follow this guide to make custom configurations and build the device service ima
 
 Configuring pre-defined devices will allow the service to automatically provision them into core-metadata. Create a list of devices with the appropriate information as outlined below.
 
-1. Make a copy of the `camera.toml.example`:  
+1. Make a copy of the `camera.yaml.example`:  
    ```bash
-   cp ./cmd/res/devices/camera.toml.example ./cmd/res/devices/camera.toml
+   cp ./cmd/res/devices/camera.yaml.example ./cmd/res/devices/camera.yaml
    ```
 
-1. Open the `cmd/res/devices/camera.toml` file using your preferred text editor and update the `Address` and `Port` fields to match the IP address of the Camera and port used for ONVIF services:
-
-   ```toml
-   [[DeviceList]]
-   Name = "Camera001"                         # Modify as desired
-   ProfileName = "onvif-camera"               # Default profile
-   Description = "onvif conformant camera"    # Modify as desired
-      [DeviceList.Protocols]
-         [DeviceList.Protocols.Onvif]
-         Address = "191.168.86.34"              # Set to your camera IP address
-         Port = "2020"                          # Set to the port your camera uses
-         SecretName = "credentials001"
-         [DeviceList.Protocols.CustomMetadata]
-         CommonName = "Outdoor camera"
+1. Open the `cmd/res/devices/camera.yaml` file using your preferred text editor and update the `Address` and `Port` fields to match the IP address of the Camera and port used for ONVIF services:
+
+   ```yaml
+   DeviceList:
+   - Name: Camera001                         # Modify as desired
+      ProfileName: onvif-camera              # Default profile
+      Description: onvif conformant camera   # Modify as desired
+      Protocols:
+         Onvif:
+         Address: 192.168.12.123              # Set to your camera IP address
+         Port: '80'                           # Set to the port your camera uses
+         FriendlyName: Home camera
+         MACAddress: 'aa:bb:cc:dd:ee:ff'
+         CustomMetadata:
+         Location: Front door
+         Color: Black and white
```

Comment:
This section still doesn't match the TOML. Also missing indentation and CustomMetadata